cmake/CompilerRuntime.cmake

```diff
@@ -39,10 +39,17 @@ macro (set_compiler_runtime TARGET_NAME)
 			# Should never happen, but if more combinations are needed, this will cover edge cases.
 			message (FATAL_ERROR "set_compiler_runtime(): unknown runtime type selected")
 		endif ()
-	else ()
+	elif (APPLE)
```

Comment:
I guess, this line causes CI failure

Reply:
Right, I forgot that it's an elseif in CMake. I haven't tested it, I only know about the flag.